package.json

```diff
@@ -11,7 +11,8 @@
     "@sentry/browser": "^4.6.6",
     "bulma": "^0.7.5",
     "bulmaswatch": "^0.7.2",
-    "hammerjs": "^2.0.8"
+    "hammerjs": "^2.0.8",
+    "material-design-lite": "^1.3.0"
```

Comment:
Nie jestem przekonany czy powinniśmy używać mdl skoro mamy już wciągniętą bulmę. Po pierwsze to nam zwiększy znacznie rozmiar bundla, po drugie wydaje mi się to niekonsekwencją.

Reply:
z tego co widzę to mdl jest użyte do ostylowania:

- textarea / text input
- przycisku załaczania pliku
- checkboxów

moim zdaniem fajnie by było (i jest to możliwe) mdl'a wywalic

Reply:
albo użyć tylko wybranych komponentów a nie całego. to chyba dam sam rade zrobić. takim jestem pro frontendem ;)

Reply:
załątwione